### PR TITLE
Make v0.5.0 release

### PR DESCRIPTION
**Pull request by ramirezdiego**

### CODE REVIEW

**Review by jorana — APPROVED** [review]

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+v0.5.0
```
Comment (jorana) [inline HISTORY.md:1]: @ramirezdiego , worth adding the data to the title of the release like in https://github.com/XENONnT/straxen/blob/master/HISTORY.md?

**Review by jorana — APPROVED** [review]

File: HISTORY.md (hunk 1 above)
Comment (ramirezdiego) [reply HISTORY.md:1]: Ok, will do. Was respecting the old format, but I guess this helps for PyPi visualization.

Comment (jorana) [reply HISTORY.md]: Many thanks, I did the same previously but only realized that I omitted it later

**Comment by petergaemers** [comment]
Thanks Diego :)

**Pull request merged**